internal/pkg/agent/cmd/container.go

```diff
@@ -701,8 +701,8 @@ func truncateString(b []byte) string {
 
 // runLegacyAPMServer extracts the bundled apm-server from elastic-agent
 // to path and runs it with args.
-func runLegacyAPMServer(streams *cli.IOStreams, path string) (*process.Info, error) {
-	name := "apm-server"
+func runLegacyAPMServer(streams *cli.IOStreams) (*process.Info, error) {
+	name := "apm"
```

Comment:
why did this change and does it have any consequences on the path, log path, etc?

Reply:
no this is for our resolution. based on this name we find correct process. while before it was a binary name now we have a mapping for input type